holoviews/plotting/bokeh/widgets.py

```diff
@@ -273,13 +275,10 @@ def _plot_figure(self, idx, fig_format='json'):
         first call and
         """
         self.plot.update(idx)
-        if self.embed or fig_format == 'html':
-            if fig_format == 'html':
```

Comment:
I assume ``fig_format == 'html'`` is still handled but somewhere else I haven't looked at yet...

Reply:
Previously it was used both for the initial frame and for the bokeh plot update messages, now the initial frame is handled separately.